app/core/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """The recipe model"""
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=500)
```

Comment:
You could also use a [TextField](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.TextField) instead

Reply:
Ok